packages/backend/src/ee/services/ai/agents/agent.ts

```diff
@@ -90,13 +93,44 @@ export const runAgent = async ({
 
     const model = openai('gpt-4.1');
 
+    console.dir(messages, { depth: null });
+
     const result = await generateText({
         model,
         tools,
         messages,
         maxSteps: 10,
         maxRetries: 3,
         temperature: 0.2,
+        experimental_repairToolCall: async ({
```

Comment:
lack of this was a regression that we were supporting in Langchain implementation.